deploy/gatekeeper.yaml

```diff
@@ -241,9 +241,9 @@ metadata:
   namespace: gatekeeper-system
 ---
 apiVersion: admissionregistration.k8s.io/v1beta1
-kind: ValidatingWebhookConfiguration
+kind: MutatingWebhookConfiguration
```

Comment:
We should keep both validating and mutating registrations... or optionally get rid of these altogether, as the process self-registers.

Reply:
Done.